ft-sdk/src/auth/schema.rs

```diff
@@ -11,18 +13,5 @@ diesel::table! {
     }
 }
 
-diesel::table! {
-    use diesel::sql_types::*;
-
-    fastn_session (id) {
-        id -> Text,
-        uid -> Nullable<Int8>,
-        data -> Text,
-        updated_at -> Timestamptz,
-        created_at -> Timestamptz,
-    }
-}
-
 diesel::joinable!(fastn_session -> fastn_user (uid));
-
 diesel::allow_tables_to_appear_in_same_query!(fastn_user, fastn_session,);
```

Comment:
```suggestion
diesel::allow_tables_to_appear_in_same_query!(fastn_user, ft_sdk::schema::fastn_session,);
```

I avoid using `use` statements.

Reply:
the diesel macro does not support `::` in names that's why a `use` is used here. This is unavoidable.